boards/arm/serpente/doc/index.rst

```diff
@@ -0,0 +1,126 @@
+.. _serpente:
+
+Arturo182 Serpente
+##################
+
+Overview
+********
+
+The Serpente is a very small low-cost development and prototyping
+board equipped with 4MB flash storage, a PWM enabled RGB led and 6 I/O pins.
```

Comment:
Note that Zephyr now has a PWM driver for the SAMD21.  Consider enabling it to drive the LED.